Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-11: To declare War, grant Letters of Marque and Reprisal, and make Rules concerning Captures on Land and Water;
+10: To provide for calling forth the Militias of the several States to execute the Laws of the Union, suppress Insurrections and repel Invasions;
 
-12: To raise and support Armies, but no Appropriation of Money to that Use shall be for a longer Term than two Years;
+11: To provide for organizing, arming, and disciplining, and for governing such Part of the Militias of the several States as may be employed in the Service of the Citizens and Residents of these United States, reserving to the States respectively, the Appointment of the Officers, and the Authority of training the Militia according to the discipline prescribed by Congress;
 
-13: To provide and maintain a Navy;
+12: To exercise exclusive Legislation in all Cases whatsoever, over such District (not exceeding ten Miles square) as may, by Cession of particular States, and the Acceptance of Congress, become the Seat of the Government of these United States, and to exercise like Authority over all Places purchased by the Consent of the Legislature of the State in which the Same shall be, for the Erection of Forts, Magazines, Arsenals, dock-Yards, and other needful Buildings;
 
-14: To make Rules for the Government and Regulation of the land and naval Forces;
-
-15: To provide for calling forth the Militia to execute the Laws of the Union, suppress Insurrections and repel Invasions;
-
-16: To provide for organizing, arming, and disciplining, the Militia, and for governing such Part of them as may be employed in the Service of the United States, reserving to the States respectively, the Appointment of the Officers, and the Authority of training the Militia according to the discipline prescribed by Congress;
-
-17: To exercise exclusive Legislation in all Cases whatsoever, over such District (not exceeding ten Miles square) as may, by Cession of particular States, and the Acceptance of Congress, become the Seat of the Government of the United States, and to exercise like Authority over all Places purchased by the Consent of the Legislature of the State in which the Same shall be, for the Erection of Forts, Magazines, Arsenals, dock-Yards, and other needful Buildings;—And
-
-18: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and all other Powers vested by this Constitution in the Government of the United States, or in any Department or Officer thereof.
+13: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and which shall be within the scope of the foregoing Powers. It shall be the duty of all Representatives and Senators, the President of these United States, all State Executives, and State Legislatures, to ensure before their approval that a Bill under consideration falls within the scope of the foregoing powers, and is not in violation of any term of this Constitution.
```

Comment:
Let this serve as one more reminder, in case a Federalist decides they want to expand the powers of the Congress: you are limited only to powers specifically prescribed and granted.

I place the duty to enforce this broadly, on every elected member of the government. The decision of Marbury v. Madison that it should be the Supreme Court's job to decide constitutional questions has given us significant problems over the years, as legislatures and executives have formed a habit of taking unconstitutional actions, and then letting them persist until a challenge reaches the Supreme Court. This dynamic has allowed a slow buildup of unconstitutional authority. Let them instead have the duty to go the other way, and let their voters hold them to account for failures in this duty.